src/models/utils.ts

```diff
@@ -370,3 +379,79 @@ export function parseNumAsTimeString(value: number): string {
 
     return tempVal > 0 ? rs + '.' + mss : rs;
 }
+
+function getConfiguration(): vscode.WorkspaceConfiguration {
+    return vscode.workspace.getConfiguration(Constants.extensionConfigSectionName);
+}
+
+export function getConfigTracingLevel(): string {
+	let config = getConfiguration();
+	if (config) {
+		return config.get(configTracingLevel);
+	}
+	else {
+		return undefined;
```

Comment:
>undefined [](start = 9, length = 9)

Not sure you want to be returning undefined for these config settings - it's unlikely the service exe's will be able to handle that correctly. I would suggest logging an error and then just not including that option at all if the config is invalid for some reason.